### PR TITLE
Fix resources

### PR DESCRIPTION
**Pull request by MadsBogeskov**
Reverts the use of swift resources

### CODE REVIEW

File: Sources/SwaggerSwift/Keywords.swift [hunk 1]
```diff
@@ -1,3 +1,10 @@
+//
+//  File.swift
+//  
+//
+//  Created by Maximilian Scheid on 30/08/2021.
+//
+
```
Comment (simonbs) [inline Sources/SwaggerSwift/Keywords.swift:7]: No changes to this file but adding a header 🧐

File: Sources/SwaggerSwift/start.swift [hunk 2]
```diff
@@ -7,6 +7,221 @@ enum HTTPMethod: String {
...
+
+let dummyTest = """
+import XCTest
+
+public struct DummyTest {
+    func testNetwork() {
+        XCTAssertTrue(true)
+    }
+}
+"""
```
Comment (simonbs) [inline Sources/SwaggerSwift/start.swift:64]: I'm probably lacking some context here but I'll go ahead anyway; why does it make sense for the code generator to create a dummy test?

**Review by simonbs — APPROVED** [review]

File: Sources/SwaggerSwift/start.swift (hunk 2 above)
Comment (mxsc) [reply Sources/SwaggerSwift/start.swift:64]: to claim that we have tests I guess 😛

Comment (MadsBogeskov) [reply Sources/SwaggerSwift/start.swift:64]: its a swift package requirement 🤷‍♂️

File: Sources/SwaggerSwift/Keywords.swift (hunk 1 above)
Comment (MadsBogeskov) [reply Sources/SwaggerSwift/Keywords.swift:7]: this just reverts code

File: Sources/SwaggerSwift/start.swift (hunk 2 above)
Comment (mxsc) [reply Sources/SwaggerSwift/start.swift:64]: it works without too

Comment (mxsc) [reply Sources/SwaggerSwift/start.swift:64]: see the code this generates: https://github.com/lunarway/SwaggerSwift/pull/18

**Pull request merged**